.github/stale.yml

```diff
@@ -0,0 +1,11 @@
+daysUntilStale: 60
+daysUntilClose: 7
+exemptLabels:
+  - pinned
+  - security
+staleLabel: stale
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
```

Comment:
```suggestion
markComment: >
  We marked this issue as stale because of inactivity. We'll close it in 7 days if you won't come back to us.
  Feel free to comment, re-open it, or create a new one and we'll continue helping.
  Truly yours, Cloud SDK Team
```